docs/source/installation.rst

```diff
@@ -0,0 +1,32 @@
+************
+Installation
+************
+
+The AiiDAlab widgets app comes pre-installed in the AiiDAlab environment.
+
+The app contains
+ * the `aiidalab-widgets-base` python package that contains the widgets (`from aiidalab_widgets_base import ...`)
```

Comment:
```suggestion
 * the `aiidalab-widgets-base` Python package that contains the widgets (`from aiidalab_widgets_base import ...`)
```
The name of the "Python" programming language should be capitalized, see for example: https://en.wikipedia.org/wiki/Python_(programming_language) .